pkgs/development/php-packages/psalm/composer.lock

```diff
@@ -675,7 +675,7 @@
                 {
                     "name": "Christian Weiske",
                     "email": "cweiske@cweiske.de",
-                    "homepage": "http://github.com/cweiske/jsonmapper/",
+                    "homepage": "https://github.com/cweiske/jsonmapper/",
```

Comment:
Can you please revert this change? This file is not meant to be manually modified and this shouldn't be touched. There's a lot of chance that touching this will have an impact since the checksum of the file would be different.

Reply:
Oops, thanks for the review